main.go

```diff
@@ -252,6 +270,28 @@ func waitForRayClusterAPIandSetupController(ctx context.Context, mgr ctrl.Manage
 	}
 }
 
+func setupAppWrapperController(mgr ctrl.Manager, cfg *config.CodeFlareOperatorConfiguration, certsReady chan struct{}) {
+	setupLog.Info("Waiting for certificate generation to complete")
+	<-certsReady
+	setupLog.Info("Certs ready")
+
+	if cfg.AppWrapper != nil && ptr.Deref(cfg.AppWrapper.Enabled, false) {
+		exitOnError(awctrl.SetupWebhooks(mgr, cfg.AppWrapper.Config), "error setting up AppWrapper webhook")
```

Comment:
No point in checking for the CRDs here.  The AppWrapper CRD is part of this operator and the Workload CRD must be installed or we would have failed during setupIndexer (see comment below).